server/db/driver/pg/orders.go

```diff
@@ -183,8 +183,8 @@ func (status pgOrderStatus) active() bool {
 
 // NewEpochOrder stores the given order with epoch status. This is equivalent to
 // StoreOrder with OrderStatusEpoch.
-func (a *Archiver) NewEpochOrder(ord order.Order, epochIdx, epochDur int64) error {
-	return a.storeOrder(ord, epochIdx, epochDur, orderStatusEpoch)
+func (a *Archiver) NewEpochOrder(ord order.Order, epochIdx, epochDur int64, epochGap int) error {
+	return a.storeOrder(ord, epochIdx, epochDur, epochGap, orderStatusEpoch)
```

Comment:
Since the DB is using `INT4`, we should use `int32` instead of `int`, at least on the `Archiver` interface.  It's better to use a specific size integer anyway since `int` varies on different platforms.